templates/partials/menu.html

```diff
@@ -1,40 +1,55 @@
-                        </li>
-                        <li class="nav-item">
-                            <h6><a id="aboutMenuButton" class="p-2 {{ menu_link_class }}" href="about.html">About</a></h6>
-                        </li>
-                    </ul>
+            <div class="offcanvas offcanvas-start collapse navbar-collapse w-100 mobile-menu" id="navbarSupportedContent">
+                <div class="offcanvas-body w-100">
+                    <div class="col-1">
+                        <a href="index.html">
+                            <img id="menu-logo" src="images/logos/marble.png" class="img-fluid ms-4 d-none d-lg-block menu-logo" />
+                        </a>
+                    </div>
+                    <div class="offcanvas-header d-flex d-md-none justify-content-end w-100">
+                        <button type="button" class="btn-close text-reset" data-bs-toggle="offcanvas" aria-label="Close"></button>
+                    </div>
+                    <div class="d-flex d-md-none {{ mobile_title_class }}"><h3>{{ mobile_title }}</h3></div>
+                    <div class="d-flex flex-column flex-lg-row menu-items-container">
+                        <ul class="navbar-nav mb-2 mb-lg-0">
+                            <li class="nav-item w-75">
+                                <h6 class="menu-items-spacing menu-items-border"><a id="homeMenuButton" class=" {{ menu_text_class }}" aria-current="page" href="index.html">Home</a></h6>
+                            </li>
+                            <li class="nav-item w-75">
+                                <h6 class="menu-items-spacing menu-items-border"><a id="technologyMenuButton" class="{{ menu_text_class }}" href="index.html#technology">Technology</a></h6>
+                            </li>
+                            <li class="nav-item w-75">
+                                <h6 class="menu-items-spacing menu-items-border"><a id="nodeMenuButton" class="{{ menu_text_class }}" href="node.html">Nodes</a></h6>
+                            </li>
+                            <li class="nav-item w-75">
+                                <h6 class="menu-items-spacing menu-items-border"><a id="tutorialsMenuButton" class="{{ menu_text_class }}" href="tutorials/index.html">Tutorials</a></h6>
+                            </li>
+                            <li class="nav-item w-75">
+                                <h6 class="menu-items-spacing menu-items-border"><a id="communityMenuButton" class="{{ menu_text_class }}" href="https://discourse.marbleclimate.com" target="_blank">Community</a></h6>
+                            </li>
+                            <li class="nav-item w-75 ">
+                                <h6 class="menu-items-spacing"><a id="aboutMenuButton" class="{{ menu_text_class }}" href="about.html">About</a></h6>
```

Comment:
Ok I see that you're not adding the underline after the last element here.

Just a suggestion... when you're doing something like this it's nicer to move the logic into the css so that if these ever get rearranged you'll always have the the border line between the elements and you don't have to move around the classes. 

Something like:

```css
    .navbar-nav .nav-item:not(:last-child) {
        border-bottom: solid;
        border-bottom-width: thin;
        border-bottom-color: var(--white);
    }
```

(instead of the `menu-items-border` class)

Reply:
Moved the menu-items-border styles to a class select